rholang/examples/let.rho

```diff
@@ -0,0 +1,23 @@
+
+new stdout(`rho:io:stdout`) in {
```

Comment:
Could you add an example with both connectives?

Reply:
Hmm. Both of which?

Reply:
`/\` & `\/`